scripts/build.py

```diff
@@ -44,9 +53,15 @@ def ensure_directory_exists(f):
         os.makedirs(d)
 
 
-def process_html(filename, target):
+def process_html(filename, target, file_hashes):
     f = open(filename, 'r')
     content = f.read()
+    for filepath, file_hash in file_hashes.iteritems():
+        if filepath.endswith('.html'):
```

Comment:
I'm not sure I understand why we skip html files. Could you explain?

If it's not obvious, maybe also add an inline comment to clarify.